luigi_pipeline/seqr_mito_loading.py

```diff
@@ -0,0 +1,109 @@
+import logging
+import sys
+import pkg_resources
+import pprint
+
+import luigi
+import hail as hl
+
+from lib.hail_tasks import HailMatrixTableTask, HailElasticSearchTask
+from lib.model.mito_mt_schema import SeqrMitoVariantSchema, SeqrMitoGenotypesSchema, SeqrMitoVariantsAndGenotypesSchema
+import seqr_loading
+
+logger = logging.getLogger(__name__)
+
+
+class SeqrMitoVariantMTTask(seqr_loading.SeqrVCFToMTTask):
```

Comment:
what is the purpose of this unoptimized task? We have the unoptimized regular loadnig task for local installs but thats out of scope for mito

Reply:
To my understanding, the purpose of the optimization is to separate the variant annotations and the genotype annotations into two tasks. It could improve performance by more parallel.
I'm not sure the unoptimized way still works or not. But I will try.

Reply:
just don't make this task at all. its not used and theres no point

Reply:
you misunderstood me. Remove the UN-optimized task - keep ONLY the optimized task

Bust also, do what I said about inheriting classes before that